bundles/org.openhab.core.io.monitor/src/main/java/org/openhab/core/io/monitor/internal/EventLogger.java

```diff
@@ -72,14 +73,8 @@ public void receive(Event event) {
         }
     }
 
-    private Logger getLogger(String eventType) {
-        String loggerName = "openhab.event." + eventType;
-        Logger logger = eventLoggers.get(loggerName);
-        if (logger == null) {
-            logger = LoggerFactory.getLogger(loggerName);
-            eventLoggers.put(loggerName, logger);
-        }
-        return logger;
+    private @Nullable Logger getLogger(String eventType) {
```

Comment:
That's not correct. It's `@NonNull`, the checker is unable to detect it due to limitations in ECJ. Please revert that and use `Objects.requireNonNull` around `eventLoggers.computeIfAbsent(...)`.

Reply:
Thanks! I didn't know how to resolve that issue.